docs/source/developer_guide/plugins/overview.rst

```diff
@@ -59,19 +55,19 @@ It is important to keep in mind that, even though there is very little that is i
 
 Failure to comply with the points below will probably mean that your plugin will not be able to be listed as an officially endorsed plugin.
 
-A plugin may / should not
+A plugin may/should not:
 
-* change the database schema AiiDA uses
-* monkey patch anything within the ``aiida`` namespace nor the namespace itself
-* use functionality of aiida or it's classes, which is marked as implementation detail, examples include functions, methods and classes starting with an underscore, '`_`'.
-* circumvent data proveniency in any way
+* change the database schema AiiDA uses;
+* monkey patch anything within the ``aiida`` namespace or the namespace itself;
+* in general, use functionality of AiiDA or its classes which is marked as implementation detail: examples include functions, methods and classes starting with an underscore symbol: `_`.
```

Comment:
either:
use a functionality of AiiDA ... which is marked
or:
use funcionalities of AiiDA ... which are marked